code/modules/mob/living/silicon/robot/emote.dm

```diff
@@ -3,7 +3,10 @@
 	emote_type = EMOTE_AUDIBLE
 
 /datum/emote/sound/silicon
-	mob_type_allowed_typecache = list(/mob/living/silicon)
+	//yogstart - Drone buffs pt 1
+	mob_type_allowed_typecache = list(/mob/living/silicon,/mob/living/simple_animal/drone)
+	mob_type_blacklist_typecache = list(/mob/living/simple_animal/drone/cogscarab)

```

Comment:
I'm pretty sure you can replicate these both to a file in the ``yogstation`` folder.